src/base-element.js

```diff
@@ -704,6 +704,22 @@ export class BaseElement {
   }
 
   /**
+   * Collapses the element, and notifies its owner (if there is one) that the
+   * element is no longer present.
```

Comment:
Was just thinking to document a bit more what the API is for and how one might use it.